lib/chai/utils/hasProperty.js

```diff
@@ -49,7 +49,7 @@ var literals = {
 };
 
 module.exports = function hasProperty(name, obj) {
-  var ot = type(obj);
+  var ot = type(obj).toLowerCase();
```

Comment:
It looks like we do not have unit testing for `hasProperty` utility.

I don't know what are the plans for this utility, is this going to a separate module?
Do you think that it's worth to add unit tests to it right now?